dali/operators/reader/numpy_reader_gpu_op.cc

```diff
@@ -102,7 +102,6 @@ void NumpyReaderGPU::Prefetch() {
                                kGDSChunkGranularity);
   chunk_size = std::min(chunk_size, curr_tensor_list.nbytes());
 
-
```

Comment:
Why touch this file at all?

Reply:
I had to adjust during rebase and then adjust it back, I guess the line was removed in the process. Do you miss it?